R/init.R

```diff
@@ -10,32 +10,29 @@
 #' End-users: This is the most important function for you to start a
 #' `teal` app that is composed of `teal` modules.
 #'
-#' @details
-#' When initializing the `teal` app, if `datanames` are not set for the `teal_data` object,
-#' defaults from the `teal_data` environment will be used.
-#'
 #' @param data (`teal_data` or `teal_data_module`)
-#' For constructing the data object, refer to [teal_data()] and [teal_data_module()].
+#'   For constructing the data object, refer to [teal_data()] and [teal_data_module()].
+#'   If `datanames` are not set for the `teal_data` object, defaults from the `teal_data` environment will be used.
 #' @param modules (`list` or `teal_modules` or `teal_module`)
-#'   nested list of `teal_modules` or `teal_module` objects or a single
+#'   Nested list of `teal_modules` or `teal_module` objects or a single
 #'   `teal_modules` or `teal_module` object. These are the specific output modules which
 #'   will be displayed in the `teal` application. See [modules()] and [module()] for
 #'   more details.
-#' @param filter (`teal_slices`)
-#'   Specifies the initial filter using [teal_slices()].
-#' @param title (`shiny.tag` or `character(1)`)
-#'   The browser window title. Defaults to a title "teal app" with the icon of NEST.
+#' @param filter (`teal_slices`) Optionally,
+#'   specifies the initial filter using [teal_slices()].
+#' @param title (`shiny.tag` or `character(1)`) Optionally,
+#'   the browser window title. Defaults to a title "teal app" with the icon of NEST.
 #'   Can be created using the `build_app_title()` or
 #'   by passing a valid `shiny.tag` which is a head tag with title and link tag.
-#' @param header (`shiny.tag` or `character(1)`)
-#'   The header of the app.
-#' @param footer (`shiny.tag` or `character(1)`)
-#'   The footer of the app.
-#' @param id (`character`) optional
-#'   string specifying the `shiny` module id in cases it is used as a `shiny` module
+#' @param header (`shiny.tag` or `character(1)`) Optionally,
+#'   the header of the app.
+#' @param footer (`shiny.tag` or `character(1)`) Optionally,
+#'   the footer of the app.
+#' @param id (`character`) Optionally,
+#'   a string specifying the `shiny` module id in cases it is used as a `shiny` module
 #'   rather than a standalone `shiny` app. This is a legacy feature.
-#' @param landing_popup (`teal_module`) optional
-#'   A `landing_popup_module` to show up as soon as the teal app is initialized.
+#' @param landing_popup (`teal_module`) Optionally,
+#'   a `landing_popup_module` to show up as soon as the teal app is initialized.
```

Comment:
FYI:
- not consistent starting with capital
- not consistent usage of "optional" for arguments with default value
- Changed "optional {text}" into "Optionally, {text}". Even though I am aware of https://github.com/insightsengineering/nestdevs-tasks/issues/52, the agreed way of doing this just looks weird (see below). I decided to apply how `dplyr` describes its optional arguments ([example](https://dplyr.tidyverse.org/reference/mutate.html#:~:text=tidy%2Dselect%3E-,Optionally,-%2C%20control%20where%20new)).
![image](https://github.com/user-attachments/assets/06981b51-137d-4ae4-bb11-995504fef094)